build/postgres/Dockerfile

```diff
@@ -12,6 +12,7 @@ RUN cd /opt && \
     USE_PGXS=1 make install
 
 COPY build/postgres/postgresql.conf /etc/postgres/postgresql.conf
+COPY build/postgres/pg_hba.conf /etc/postgres/pg_hba.conf
 RUN chown -R postgres:postgres /etc/postgres
 
-CMD ["postgres", "-c", "config_file=/etc/postgres/postgresql.conf"]
+CMD ["postgres", "-cconfig_file=/etc/postgres/postgresql.conf", "-chba_file=/etc/postgres/pg_hba.conf"]
```

Comment:
Huh, surprised this works for the `--command` option of `postgres`. Normally I would expect a space between `-c` and the command/variable being applied, like the following:

```
CMD ["postgres", "-c", "config_file=/etc/postgres/postgresql.conf", "-c", "hba_file=/etc/postgres/pg_hba.conf"]
```

Reply:
it works both ways, I figured abbreviating would keep things tidy with `one array item == one config option`
happy to switch it back but it's working

Reply:
It's fine as-is 👍